[INFO] [2020-11-20 07:07:11,963] [1:140008512935680] - job_controller.py[line:52]: job parameters:{'job_type': 'train', 'work_mode': 1, 'backend': 0, 'computing_engine': 'EGGROLL', 'federation_engine': 'EGGROLL', 'storage_engine': 'EGGROLL', 'engines_address': {}, 'federated_mode': 'MULTIPLE', 'task_parallelism': 1, 'computing_partitions': 4, 'federated_status_collect_type': 'PUSH', 'model_id': 'guest-10000#host-9999#model', 'model_version': '202011200707112994524', 'eggroll_run': {}, 'spark_run': {}, 'rabbitmq_run': {}, 'adaptation_parameters': {'task_nodes': 1, 'task_cores_per_node': 4, 'task_memory_per_node': 0}}
[INFO] [2020-11-20 07:07:14,809] [1:140008512935680] - resource_manager.py[line:172]: apply job 202011200707112994524 resource(cores 4 memory 0) on host 9999 successfully, remaining cores: 16 remaining memory: 0
[INFO] [2020-11-20 07:07:15,290] [1:140008262174464] - job_controller.py[line:238]: try to start job 202011200707112994524 on host 9999
[INFO] [2020-11-20 07:07:15,291] [1:140008262174464] - job_controller.py[line:247]: extra info: {'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:07:15,292] [1:140008262174464] - job_saver.py[line:43]: try to update job 202011200707112994524 status to running
[INFO] [2020-11-20 07:07:15,460] [1:140008262174464] - job_saver.py[line:46]: update job 202011200707112994524 status successfully
[INFO] [2020-11-20 07:07:15,461] [1:140008262174464] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:07:15,552] [1:140008262174464] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': 9999, 'status': 'running', 'start_time': 1605856035291, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:07:15,552] [1:140008262174464] - job_controller.py[line:251]: start job 202011200707112994524 on host 9999 successfully
[INFO] [2020-11-20 07:07:16,397] [1:140008262174464] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:16,449] [1:140008262174464] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:07:17,576] [1:140008680724224] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_reader_1 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:07:17,690] [1:140008680724224] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:07:17,691] [1:140008680724224] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_reader_1 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:07:17,691] [1:140008680724224] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n reader_1 -t 202011200707112994524_reader_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/reader_1/202011200707112994524_reader_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:07:17,816] [1:140008680724224] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n reader_1 -t 202011200707112994524_reader_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/reader_1/202011200707112994524_reader_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 18802
[INFO] [2020-11-20 07:07:17,819] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[INFO] [2020-11-20 07:07:17,979] [1:140008680724224] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update successfully
[INFO] [2020-11-20 07:07:18,434] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:18,502] [1:140008680724224] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856037819}
[INFO] [2020-11-20 07:07:19,010] [1:140008680724224] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_reader_1 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:07:25,257] [18802:139627058091840] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:07:25,258] [18802:139627058091840] - task_executor.py[line:57]: Namespace(component_name='reader_1', config='/data/projects/fate/jobs/202011200707112994524/host/9999/reader_1/202011200707112994524_reader_1/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_reader_1', task_version=0)
[INFO] [2020-11-20 07:07:25,773] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[INFO] [2020-11-20 07:07:25,829] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update successfully
[INFO] [2020-11-20 07:07:26,365] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:26,396] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'reader_1', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 18802, 'party_status': 'running'}
[INFO] [2020-11-20 07:07:50,295] [1:140008262174464] - job_tracker.py[line:97]: save job 202011200707112994524 component reader_1 on host 9999 reader_namespace reader_name metric meta
[INFO] [2020-11-20 07:07:50,456] [18802:139627058091840] - job_tracker.py[line:159]: task id 202011200707112994524_reader_1 output data table is none
[INFO] [2020-11-20 07:07:50,636] [1:140008262174464] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[INFO] [2020-11-20 07:07:50,699] [1:140008262174464] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update successfully
[INFO] [2020-11-20 07:07:51,175] [1:140008262174464] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:51,233] [1:140008262174464] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'reader_1', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 18802, 'party_status': 'success', 'end_time': 1605856070457, 'elapsed': 25199}
[INFO] [2020-11-20 07:07:51,844] [1:140008262174464] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[WARNING] [2020-11-20 07:07:51,884] [1:140008262174464] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update does not take effect
[INFO] [2020-11-20 07:07:52,273] [1:140008262174464] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:52,320] [1:140008262174464] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'reader_1', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 18802, 'party_status': 'success', 'end_time': 1605856070457, 'elapsed': 25199}
[INFO] [2020-11-20 07:07:53,292] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:07:53,366] [1:140008680724224] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:07:53,479] [1:140008680724224] - resource_manager.py[line:271]: task 202011200707112994524_reader_1 0 return resource successfully
[INFO] [2020-11-20 07:07:53,517] [1:140008680724224] - job_tracker.py[line:424]: clean task 202011200707112994524_reader_1 0 on host 9999
[INFO] [2020-11-20 07:07:59,493] [1:140008680724224] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_reader_1_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:07:59,559] [1:140008680724224] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_reader_1_0 on host 9999 done
[INFO] [2020-11-20 07:08:07,150] [1:140008680724224] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_reader_1 host 9999 with success party status process pid:18802
[INFO] [2020-11-20 07:08:07,153] [1:140008680724224] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:08:07,154] [1:140008680724224] - job_utils.py[line:382]: this pid 18802 is not job 202011200707112994524 task 202011200707112994524_reader_1 host 9999 executor
[INFO] [2020-11-20 07:08:07,180] [1:140008680724224] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_reader_1_0_host_9999
[INFO] [2020-11-20 07:08:07,181] [1:140008680724224] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:08:07,220] [1:140008680724224] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 19220
[INFO] [2020-11-20 07:08:07,223] [1:140008680724224] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_reader_1 0 on host 9999 process 18802 kill success
[INFO] [2020-11-20 07:08:07,224] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:08:07,265] [1:140008680724224] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:08:08,160] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[WARNING] [2020-11-20 07:08:08,619] [1:140008680724224] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update does not take effect
[INFO] [2020-11-20 07:08:09,995] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:10,047] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:08:11,171] [1:140008680724224] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_reader_0 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:08:11,193] [1:140008680724224] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:08:11,194] [1:140008680724224] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_reader_0 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:08:11,195] [1:140008680724224] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n reader_0 -t 202011200707112994524_reader_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/reader_0/202011200707112994524_reader_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:08:11,233] [1:140008680724224] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n reader_0 -t 202011200707112994524_reader_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/reader_0/202011200707112994524_reader_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 19259
[INFO] [2020-11-20 07:08:11,236] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[INFO] [2020-11-20 07:08:11,424] [1:140008680724224] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update successfully
[INFO] [2020-11-20 07:08:12,203] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:12,266] [1:140008680724224] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856091236}
[INFO] [2020-11-20 07:08:12,830] [1:140008680724224] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_reader_0 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:08:12,883] [19220:140634401261376] - session_utils.py[line:38]: start stop session 202011200707112994524_reader_1_0_host_9999
[INFO] [2020-11-20 07:08:13,817] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:08:14,136] [1:140008722671360] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:16,495] [19220:140634401261376] - session_utils.py[line:47]: stop session 202011200707112994524_reader_1_0_host_9999 success
[INFO] [2020-11-20 07:08:17,130] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:17,201] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:18,537] [19259:140634121942848] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:08:18,538] [19259:140634121942848] - task_executor.py[line:57]: Namespace(component_name='reader_0', config='/data/projects/fate/jobs/202011200707112994524/host/9999/reader_0/202011200707112994524_reader_0/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_reader_0', task_version=0)
[INFO] [2020-11-20 07:08:19,067] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[INFO] [2020-11-20 07:08:19,173] [1:140008680724224] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update successfully
[INFO] [2020-11-20 07:08:19,615] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:19,645] [1:140008680724224] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'reader_0', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19259, 'party_status': 'running'}
[INFO] [2020-11-20 07:08:20,275] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:20,324] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:23,610] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:23,677] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:26,762] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:26,813] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:29,972] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:30,030] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:33,044] [1:140008680724224] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:33,114] [1:140008680724224] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:35,926] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:35,970] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:38,973] [1:140008680724224] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:39,067] [1:140008680724224] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:41,725] [1:140008680724224] - job_tracker.py[line:97]: save job 202011200707112994524 component reader_0 on host 9999 reader_namespace reader_name metric meta
[INFO] [2020-11-20 07:08:41,823] [19259:140634121942848] - job_tracker.py[line:159]: task id 202011200707112994524_reader_0 output data table is none
[INFO] [2020-11-20 07:08:41,840] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[INFO] [2020-11-20 07:08:41,906] [1:140008680724224] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update successfully
[INFO] [2020-11-20 07:08:42,063] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:08:42,222] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 12.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:08:42,604] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:42,661] [1:140008680724224] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'reader_0', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19259, 'party_status': 'success', 'end_time': 1605856121824, 'elapsed': 23285}
[INFO] [2020-11-20 07:08:43,133] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[WARNING] [2020-11-20 07:08:43,210] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update does not take effect
[INFO] [2020-11-20 07:08:43,579] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:43,604] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'reader_0', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19259, 'party_status': 'success', 'end_time': 1605856121824, 'elapsed': 23285}
[INFO] [2020-11-20 07:08:45,091] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:45,155] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:08:45,222] [1:140008554882816] - resource_manager.py[line:271]: task 202011200707112994524_reader_0 0 return resource successfully
[INFO] [2020-11-20 07:08:45,278] [1:140008554882816] - job_tracker.py[line:424]: clean task 202011200707112994524_reader_0 0 on host 9999
[INFO] [2020-11-20 07:08:51,374] [1:140008554882816] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_reader_0_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:08:51,474] [1:140008554882816] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_reader_0_0 on host 9999 done
[INFO] [2020-11-20 07:08:58,221] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_reader_0 host 9999 with success party status process pid:19259
[INFO] [2020-11-20 07:08:58,224] [1:140008512935680] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:08:58,225] [1:140008512935680] - job_utils.py[line:382]: this pid 19259 is not job 202011200707112994524 task 202011200707112994524_reader_0 host 9999 executor
[INFO] [2020-11-20 07:08:58,241] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_reader_0_0_host_9999
[INFO] [2020-11-20 07:08:58,243] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:08:58,282] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 19479
[INFO] [2020-11-20 07:08:58,285] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_reader_0 0 on host 9999 process 19259 kill success
[INFO] [2020-11-20 07:08:58,287] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:08:58,328] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:08:58,742] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[WARNING] [2020-11-20 07:08:58,795] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update does not take effect
[INFO] [2020-11-20 07:08:59,967] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:00,050] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:09:01,020] [1:140008512935680] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_dataio_0 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:09:01,126] [1:140008512935680] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:09:01,127] [1:140008512935680] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:09:01,128] [1:140008512935680] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n dataio_0 -t 202011200707112994524_dataio_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/dataio_0/202011200707112994524_dataio_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:09:01,164] [1:140008512935680] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n dataio_0 -t 202011200707112994524_dataio_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/dataio_0/202011200707112994524_dataio_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 19507
[INFO] [2020-11-20 07:09:01,167] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[INFO] [2020-11-20 07:09:01,280] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update successfully
[INFO] [2020-11-20 07:09:01,710] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:01,864] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856141167}
[INFO] [2020-11-20 07:09:02,429] [1:140008512935680] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:09:03,084] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:09:03,158] [1:140008512935680] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 25.0, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:09:03,805] [19479:139921768294208] - session_utils.py[line:38]: start stop session 202011200707112994524_reader_0_0_host_9999
[INFO] [2020-11-20 07:09:07,123] [19479:139921768294208] - session_utils.py[line:47]: stop session 202011200707112994524_reader_0_0_host_9999 success
[INFO] [2020-11-20 07:09:08,749] [19507:140606740244288] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:09:08,749] [19507:140606740244288] - task_executor.py[line:57]: Namespace(component_name='dataio_0', config='/data/projects/fate/jobs/202011200707112994524/host/9999/dataio_0/202011200707112994524_dataio_0/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_dataio_0', task_version=0)
[INFO] [2020-11-20 07:09:09,206] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[INFO] [2020-11-20 07:09:09,289] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update successfully
[INFO] [2020-11-20 07:09:09,693] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:09,725] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'dataio_0', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19507, 'party_status': 'running'}
[INFO] [2020-11-20 07:09:22,474] [19507:140606740244288] - job_tracker.py[line:171]: Get task 202011200707112994524_reader_0 0 output table experiment breast_hetero_host
[INFO] [2020-11-20 07:09:32,600] [19507:140606740244288] - job_tracker.py[line:124]: persisting the component output temporary table to output_data_202011200707112994524_dataio_0_0 56c960f42aff11ebaabe3ea3df35c0a6
[INFO] [2020-11-20 07:09:32,600] [19507:140606740244288] - job_tracker.py[line:126]: output data table partitions is 4
[INFO] [2020-11-20 07:09:33,589] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[INFO] [2020-11-20 07:09:33,729] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update successfully
[INFO] [2020-11-20 07:09:34,127] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:34,186] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'dataio_0', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19507, 'party_status': 'success', 'end_time': 1605856173555, 'elapsed': 24805}
[INFO] [2020-11-20 07:09:34,708] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[WARNING] [2020-11-20 07:09:34,755] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update does not take effect
[INFO] [2020-11-20 07:09:35,143] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:35,218] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'dataio_0', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19507, 'party_status': 'success', 'end_time': 1605856173555, 'elapsed': 24805}
[INFO] [2020-11-20 07:09:36,449] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:36,524] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:09:36,595] [1:140008554882816] - resource_manager.py[line:271]: task 202011200707112994524_dataio_0 0 return resource successfully
[INFO] [2020-11-20 07:09:36,624] [1:140008554882816] - job_tracker.py[line:424]: clean task 202011200707112994524_dataio_0 0 on host 9999
[INFO] [2020-11-20 07:09:42,652] [1:140008554882816] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_dataio_0_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:09:42,746] [1:140008554882816] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_dataio_0_0 on host 9999 done
[INFO] [2020-11-20 07:09:49,758] [1:140008722671360] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_dataio_0 host 9999 with success party status process pid:19507
[INFO] [2020-11-20 07:09:49,763] [1:140008722671360] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:09:49,763] [1:140008722671360] - job_utils.py[line:382]: this pid 19507 is not job 202011200707112994524 task 202011200707112994524_dataio_0 host 9999 executor
[INFO] [2020-11-20 07:09:49,794] [1:140008722671360] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_dataio_0_0_host_9999
[INFO] [2020-11-20 07:09:49,795] [1:140008722671360] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:09:49,829] [1:140008722671360] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 19707
[INFO] [2020-11-20 07:09:49,834] [1:140008722671360] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 on host 9999 process 19507 kill success
[INFO] [2020-11-20 07:09:49,835] [1:140008722671360] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:09:49,866] [1:140008722671360] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:09:50,213] [1:140008722671360] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[WARNING] [2020-11-20 07:09:50,247] [1:140008722671360] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update does not take effect
[INFO] [2020-11-20 07:09:51,504] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:09:51,588] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:09:52,830] [1:140008554882816] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_dataio_1 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:09:52,874] [1:140008554882816] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:09:52,875] [1:140008554882816] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:09:52,876] [1:140008554882816] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n dataio_1 -t 202011200707112994524_dataio_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/dataio_1/202011200707112994524_dataio_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:09:53,000] [1:140008554882816] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n dataio_1 -t 202011200707112994524_dataio_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/dataio_1/202011200707112994524_dataio_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 19734
[INFO] [2020-11-20 07:09:53,003] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[INFO] [2020-11-20 07:09:53,245] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update successfully
[INFO] [2020-11-20 07:09:53,909] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:09:53,968] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856193003}
[INFO] [2020-11-20 07:09:54,486] [1:140008554882816] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:09:55,130] [19707:139822796060480] - session_utils.py[line:38]: start stop session 202011200707112994524_dataio_0_0_host_9999
[INFO] [2020-11-20 07:09:55,158] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:09:55,249] [1:140008554882816] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:09:58,611] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:09:58,724] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:00,328] [19734:140323371067200] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:10:00,329] [19734:140323371067200] - task_executor.py[line:57]: Namespace(component_name='dataio_1', config='/data/projects/fate/jobs/202011200707112994524/host/9999/dataio_1/202011200707112994524_dataio_1/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_dataio_1', task_version=0)
[INFO] [2020-11-20 07:10:00,401] [19707:139822796060480] - session_utils.py[line:47]: stop session 202011200707112994524_dataio_0_0_host_9999 success
[INFO] [2020-11-20 07:10:00,904] [1:140008722671360] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[INFO] [2020-11-20 07:10:01,115] [1:140008722671360] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update successfully
[INFO] [2020-11-20 07:10:01,545] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:01,591] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:01,662] [1:140008722671360] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:10:01,689] [1:140008722671360] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'dataio_1', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19734, 'party_status': 'running'}
[INFO] [2020-11-20 07:10:04,960] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:05,134] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:08,047] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:08,087] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:11,107] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:11,162] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:12,930] [19734:140323371067200] - job_tracker.py[line:171]: Get task 202011200707112994524_reader_1 0 output table experiment breast_hetero_host
[INFO] [2020-11-20 07:10:14,554] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:14,606] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:17,820] [1:140008722671360] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:18,217] [1:140008722671360] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:21,285] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:21,330] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:22,829] [19734:140323371067200] - job_tracker.py[line:124]: persisting the component output temporary table to output_data_202011200707112994524_dataio_1_0 74b9bfd22aff11eb8dc13ea3df35c0a6
[INFO] [2020-11-20 07:10:22,830] [19734:140323371067200] - job_tracker.py[line:126]: output data table partitions is 4
[INFO] [2020-11-20 07:10:24,252] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:10:24,289] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 37.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:25,651] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[INFO] [2020-11-20 07:10:25,696] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update successfully
[INFO] [2020-11-20 07:10:26,095] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:10:26,152] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'dataio_1', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19734, 'party_status': 'success', 'end_time': 1605856225602, 'elapsed': 25272}
[INFO] [2020-11-20 07:10:26,656] [1:140008722671360] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[WARNING] [2020-11-20 07:10:26,780] [1:140008722671360] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update does not take effect
[INFO] [2020-11-20 07:10:27,102] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:10:27,157] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:10:27,225] [1:140008722671360] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:10:27,229] [1:140008512935680] - resource_manager.py[line:271]: task 202011200707112994524_dataio_1 0 return resource successfully
[INFO] [2020-11-20 07:10:27,261] [1:140008722671360] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'dataio_1', 'task_id': '202011200707112994524_dataio_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19734, 'party_status': 'success', 'end_time': 1605856225602, 'elapsed': 25272}
[INFO] [2020-11-20 07:10:27,369] [1:140008512935680] - job_tracker.py[line:424]: clean task 202011200707112994524_dataio_1 0 on host 9999
[INFO] [2020-11-20 07:10:33,794] [1:140008512935680] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_dataio_1_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:10:33,878] [1:140008512935680] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_dataio_1_0 on host 9999 done
[INFO] [2020-11-20 07:10:40,763] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_dataio_1 host 9999 with success party status process pid:19734
[INFO] [2020-11-20 07:10:40,765] [1:140008512935680] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:10:40,766] [1:140008512935680] - job_utils.py[line:382]: this pid 19734 is not job 202011200707112994524 task 202011200707112994524_dataio_1 host 9999 executor
[INFO] [2020-11-20 07:10:40,784] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_dataio_1_0_host_9999
[INFO] [2020-11-20 07:10:40,786] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:10:40,825] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 19944
[INFO] [2020-11-20 07:10:40,828] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 on host 9999 process 19734 kill success
[INFO] [2020-11-20 07:10:40,829] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:10:40,867] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:10:41,708] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[WARNING] [2020-11-20 07:10:41,735] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update does not take effect
[INFO] [2020-11-20 07:10:43,004] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:10:43,085] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:10:44,082] [1:140008554882816] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_intersection_1 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:10:44,100] [1:140008554882816] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:10:44,101] [1:140008554882816] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:10:44,101] [1:140008554882816] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n intersection_1 -t 202011200707112994524_intersection_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/intersection_1/202011200707112994524_intersection_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:10:44,132] [1:140008554882816] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n intersection_1 -t 202011200707112994524_intersection_1 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/intersection_1/202011200707112994524_intersection_1/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 19973
[INFO] [2020-11-20 07:10:44,135] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[INFO] [2020-11-20 07:10:44,196] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update successfully
[INFO] [2020-11-20 07:10:44,620] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:10:44,761] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856244135}
[INFO] [2020-11-20 07:10:45,201] [1:140008554882816] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:10:45,906] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:10:45,971] [1:140008554882816] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 50.0, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:10:46,121] [19944:140541730903872] - session_utils.py[line:38]: start stop session 202011200707112994524_dataio_1_0_host_9999
[INFO] [2020-11-20 07:10:49,738] [19944:140541730903872] - session_utils.py[line:47]: stop session 202011200707112994524_dataio_1_0_host_9999 success
[INFO] [2020-11-20 07:10:51,490] [19973:139632274605888] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:10:51,492] [19973:139632274605888] - task_executor.py[line:57]: Namespace(component_name='intersection_1', config='/data/projects/fate/jobs/202011200707112994524/host/9999/intersection_1/202011200707112994524_intersection_1/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_intersection_1', task_version=0)
[INFO] [2020-11-20 07:10:51,997] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[INFO] [2020-11-20 07:10:52,089] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update successfully
[INFO] [2020-11-20 07:10:52,470] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:10:52,506] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'intersection_1', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19973, 'party_status': 'running'}
[INFO] [2020-11-20 07:11:03,784] [19973:139632274605888] - job_tracker.py[line:171]: Get task 202011200707112994524_dataio_1 0 output table output_data_202011200707112994524_dataio_1_0 74b9bfd22aff11eb8dc13ea3df35c0a6
[INFO] [2020-11-20 07:11:18,982] [1:140008554882816] - job_tracker.py[line:74]: save job 202011200707112994524 component intersection_1 on host 9999 train intersection metric data
[INFO] [2020-11-20 07:11:19,060] [1:140008554882816] - job_tracker.py[line:97]: save job 202011200707112994524 component intersection_1 on host 9999 train intersection metric meta
[INFO] [2020-11-20 07:11:19,901] [19973:139632274605888] - job_tracker.py[line:124]: persisting the component output temporary table to output_data_202011200707112994524_intersection_1_0 96be3f402aff11eb99d93ea3df35c0a6
[INFO] [2020-11-20 07:11:19,901] [19973:139632274605888] - job_tracker.py[line:126]: output data table partitions is 4
[INFO] [2020-11-20 07:11:21,194] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[INFO] [2020-11-20 07:11:21,245] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update successfully
[INFO] [2020-11-20 07:11:21,602] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:11:21,651] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'intersection_1', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19973, 'party_status': 'success', 'end_time': 1605856281171, 'elapsed': 29678}
[INFO] [2020-11-20 07:11:22,155] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[WARNING] [2020-11-20 07:11:22,306] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update does not take effect
[INFO] [2020-11-20 07:11:22,740] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:11:22,765] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'intersection_1', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 19973, 'party_status': 'success', 'end_time': 1605856281171, 'elapsed': 29678}
[INFO] [2020-11-20 07:11:24,608] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:11:24,751] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_1', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:11:24,799] [1:140008512935680] - resource_manager.py[line:271]: task 202011200707112994524_intersection_1 0 return resource successfully
[INFO] [2020-11-20 07:11:24,829] [1:140008512935680] - job_tracker.py[line:424]: clean task 202011200707112994524_intersection_1 0 on host 9999
[INFO] [2020-11-20 07:11:31,265] [1:140008512935680] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_intersection_1_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:11:31,399] [1:140008512935680] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_intersection_1_0 on host 9999 done
[INFO] [2020-11-20 07:11:38,510] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_intersection_1 host 9999 with success party status process pid:19973
[INFO] [2020-11-20 07:11:38,512] [1:140008512935680] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:11:38,514] [1:140008512935680] - job_utils.py[line:382]: this pid 19973 is not job 202011200707112994524 task 202011200707112994524_intersection_1 host 9999 executor
[INFO] [2020-11-20 07:11:38,531] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_intersection_1_0_host_9999
[INFO] [2020-11-20 07:11:38,533] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:11:38,568] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 20215
[INFO] [2020-11-20 07:11:38,572] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 on host 9999 process 19973 kill success
[INFO] [2020-11-20 07:11:38,573] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:11:38,611] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:11:39,216] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[WARNING] [2020-11-20 07:11:39,236] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update does not take effect
[INFO] [2020-11-20 07:11:40,401] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:11:40,447] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:11:41,949] [1:140008512935680] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_intersection_0 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:11:41,964] [1:140008512935680] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:11:41,965] [1:140008512935680] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:11:41,966] [1:140008512935680] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n intersection_0 -t 202011200707112994524_intersection_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/intersection_0/202011200707112994524_intersection_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:11:42,003] [1:140008512935680] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n intersection_0 -t 202011200707112994524_intersection_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/intersection_0/202011200707112994524_intersection_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 20244
[INFO] [2020-11-20 07:11:42,005] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[INFO] [2020-11-20 07:11:42,074] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update successfully
[INFO] [2020-11-20 07:11:42,475] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:11:42,545] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856302005}
[INFO] [2020-11-20 07:11:42,984] [1:140008512935680] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:11:43,689] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:11:43,763] [1:140008512935680] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:11:44,334] [20215:140328347649856] - session_utils.py[line:38]: start stop session 202011200707112994524_intersection_1_0_host_9999
[INFO] [2020-11-20 07:11:46,880] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:11:46,935] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:11:47,710] [20215:140328347649856] - session_utils.py[line:47]: stop session 202011200707112994524_intersection_1_0_host_9999 success
[INFO] [2020-11-20 07:11:49,441] [20244:139871588984640] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:11:49,443] [20244:139871588984640] - task_executor.py[line:57]: Namespace(component_name='intersection_0', config='/data/projects/fate/jobs/202011200707112994524/host/9999/intersection_0/202011200707112994524_intersection_0/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_intersection_0', task_version=0)
[INFO] [2020-11-20 07:11:49,811] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:11:49,867] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:11:50,056] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[INFO] [2020-11-20 07:11:50,194] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update successfully
[INFO] [2020-11-20 07:11:50,602] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:11:50,629] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'intersection_0', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20244, 'party_status': 'running'}
[INFO] [2020-11-20 07:11:53,036] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:11:53,093] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:11:56,097] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:11:56,166] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:11:59,175] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:11:59,222] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:02,104] [20244:139871588984640] - job_tracker.py[line:171]: Get task 202011200707112994524_dataio_0 0 output table output_data_202011200707112994524_dataio_0_0 56c960f42aff11ebaabe3ea3df35c0a6
[INFO] [2020-11-20 07:12:02,150] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:02,234] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:05,064] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:05,106] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:08,043] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:08,172] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:11,081] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:11,138] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:14,074] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:14,121] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:17,098] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:17,145] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:17,813] [1:140008554882816] - job_tracker.py[line:74]: save job 202011200707112994524 component intersection_0 on host 9999 train intersection metric data
[INFO] [2020-11-20 07:12:17,889] [1:140008554882816] - job_tracker.py[line:97]: save job 202011200707112994524 component intersection_0 on host 9999 train intersection metric meta
[INFO] [2020-11-20 07:12:18,703] [20244:139871588984640] - job_tracker.py[line:124]: persisting the component output temporary table to output_data_202011200707112994524_intersection_0_0 b9cac0762aff11eba0823ea3df35c0a6
[INFO] [2020-11-20 07:12:18,704] [20244:139871588984640] - job_tracker.py[line:126]: output data table partitions is 4
[INFO] [2020-11-20 07:12:19,452] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[INFO] [2020-11-20 07:12:19,507] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update successfully
[INFO] [2020-11-20 07:12:19,978] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:12:20,080] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'intersection_0', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20244, 'party_status': 'success', 'end_time': 1605856339427, 'elapsed': 29984}
[INFO] [2020-11-20 07:12:20,269] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:12:20,360] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 62.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:20,944] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[WARNING] [2020-11-20 07:12:20,999] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update does not take effect
[INFO] [2020-11-20 07:12:21,456] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:12:21,494] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'intersection_0', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20244, 'party_status': 'success', 'end_time': 1605856339427, 'elapsed': 29984}
[INFO] [2020-11-20 07:12:23,410] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:12:23,464] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:12:23,556] [1:140008554882816] - resource_manager.py[line:271]: task 202011200707112994524_intersection_0 0 return resource successfully
[INFO] [2020-11-20 07:12:23,595] [1:140008554882816] - job_tracker.py[line:424]: clean task 202011200707112994524_intersection_0 0 on host 9999
[INFO] [2020-11-20 07:12:29,991] [1:140008554882816] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_intersection_0_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:12:30,156] [1:140008554882816] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_intersection_0_0 on host 9999 done
[INFO] [2020-11-20 07:12:37,213] [1:140008554882816] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_intersection_0 host 9999 with success party status process pid:20244
[INFO] [2020-11-20 07:12:37,215] [1:140008554882816] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:12:37,216] [1:140008554882816] - job_utils.py[line:382]: this pid 20244 is not job 202011200707112994524 task 202011200707112994524_intersection_0 host 9999 executor
[INFO] [2020-11-20 07:12:37,239] [1:140008554882816] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_intersection_0_0_host_9999
[INFO] [2020-11-20 07:12:37,242] [1:140008554882816] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:12:37,285] [1:140008554882816] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 20498
[INFO] [2020-11-20 07:12:37,288] [1:140008554882816] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 on host 9999 process 20244 kill success
[INFO] [2020-11-20 07:12:37,288] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:12:37,321] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:12:37,867] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[WARNING] [2020-11-20 07:12:37,920] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update does not take effect
[INFO] [2020-11-20 07:12:39,275] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:12:39,389] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:12:40,439] [1:140008554882816] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:12:40,470] [1:140008554882816] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:12:40,471] [1:140008554882816] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:12:40,471] [1:140008554882816] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n hetero_secure_boost_0 -t 202011200707112994524_hetero_secure_boost_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/hetero_secure_boost_0/202011200707112994524_hetero_secure_boost_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:12:40,508] [1:140008554882816] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n hetero_secure_boost_0 -t 202011200707112994524_hetero_secure_boost_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/hetero_secure_boost_0/202011200707112994524_hetero_secure_boost_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 20527
[INFO] [2020-11-20 07:12:40,511] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[INFO] [2020-11-20 07:12:40,569] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update successfully
[INFO] [2020-11-20 07:12:40,991] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:12:41,097] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856360511}
[INFO] [2020-11-20 07:12:41,592] [1:140008554882816] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:12:42,194] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:12:42,276] [1:140008554882816] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 75.0, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:12:42,899] [20498:139942322915136] - session_utils.py[line:38]: start stop session 202011200707112994524_intersection_0_0_host_9999
[INFO] [2020-11-20 07:12:46,424] [20498:139942322915136] - session_utils.py[line:47]: stop session 202011200707112994524_intersection_0_0_host_9999 success
[INFO] [2020-11-20 07:12:48,048] [20527:139871606757184] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:12:48,049] [20527:139871606757184] - task_executor.py[line:57]: Namespace(component_name='hetero_secure_boost_0', config='/data/projects/fate/jobs/202011200707112994524/host/9999/hetero_secure_boost_0/202011200707112994524_hetero_secure_boost_0/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_hetero_secure_boost_0', task_version=0)
[INFO] [2020-11-20 07:12:48,514] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[INFO] [2020-11-20 07:12:48,597] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update successfully
[INFO] [2020-11-20 07:12:49,400] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:12:49,427] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'hetero_secure_boost_0', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20527, 'party_status': 'running'}
[INFO] [2020-11-20 07:13:00,803] [20527:139871606757184] - job_tracker.py[line:171]: Get task 202011200707112994524_intersection_0 0 output table output_data_202011200707112994524_intersection_0_0 b9cac0762aff11eba0823ea3df35c0a6
[INFO] [2020-11-20 07:13:01,430] [20527:139871606757184] - job_tracker.py[line:171]: Get task 202011200707112994524_intersection_1 0 output table output_data_202011200707112994524_intersection_1_0 96be3f402aff11eb99d93ea3df35c0a6
[INFO] [2020-11-20 07:17:02,009] [20527:139871606757184] - job_tracker.py[line:159]: task id 202011200707112994524_hetero_secure_boost_0 output data table is none
[INFO] [2020-11-20 07:17:02,088] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[INFO] [2020-11-20 07:17:02,269] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update successfully
[INFO] [2020-11-20 07:17:02,861] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:17:02,930] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'hetero_secure_boost_0', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20527, 'party_status': 'success', 'end_time': 1605856622064, 'elapsed': 254014}
[INFO] [2020-11-20 07:17:03,420] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[WARNING] [2020-11-20 07:17:03,492] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update does not take effect
[INFO] [2020-11-20 07:17:03,937] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:17:03,978] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'hetero_secure_boost_0', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 20527, 'party_status': 'success', 'end_time': 1605856622064, 'elapsed': 254014}
[INFO] [2020-11-20 07:17:10,255] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:17:10,305] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:17:10,334] [1:140008554882816] - resource_manager.py[line:271]: task 202011200707112994524_hetero_secure_boost_0 0 return resource successfully
[INFO] [2020-11-20 07:17:10,372] [1:140008554882816] - job_tracker.py[line:424]: clean task 202011200707112994524_hetero_secure_boost_0 0 on host 9999
[INFO] [2020-11-20 07:17:16,765] [1:140008554882816] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_hetero_secure_boost_0_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:17:16,983] [1:140008554882816] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_hetero_secure_boost_0_0 on host 9999 done
[INFO] [2020-11-20 07:17:24,933] [1:140008554882816] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 host 9999 with success party status process pid:20527
[INFO] [2020-11-20 07:17:24,936] [1:140008554882816] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:17:24,937] [1:140008554882816] - job_utils.py[line:382]: this pid 20527 is not job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 host 9999 executor
[INFO] [2020-11-20 07:17:24,979] [1:140008554882816] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_hetero_secure_boost_0_0_host_9999
[INFO] [2020-11-20 07:17:24,980] [1:140008554882816] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_hetero_secure_boost_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:17:25,016] [1:140008554882816] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_hetero_secure_boost_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22498
[INFO] [2020-11-20 07:17:25,020] [1:140008554882816] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 on host 9999 process 20527 kill success
[INFO] [2020-11-20 07:17:25,021] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:17:25,057] [1:140008554882816] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:17:25,477] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[WARNING] [2020-11-20 07:17:25,516] [1:140008554882816] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update does not take effect
[INFO] [2020-11-20 07:17:26,834] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:27,030] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'running'}
[INFO] [2020-11-20 07:17:28,182] [1:140008554882816] - task_controller.py[line:66]: try to start job 202011200707112994524 task 202011200707112994524_evaluation_0 0 on host 9999 executor subprocess
[INFO] [2020-11-20 07:17:28,202] [1:140008554882816] - task_controller.py[line:85]: use computing engine EGGROLL
[INFO] [2020-11-20 07:17:28,202] [1:140008554882816] - task_controller.py[line:139]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 on host 9999 executor subprocess is ready
[INFO] [2020-11-20 07:17:28,203] [1:140008554882816] - job_utils.py[line:288]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n evaluation_0 -t 202011200707112994524_evaluation_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/evaluation_0/202011200707112994524_evaluation_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380
[INFO] [2020-11-20 07:17:28,234] [1:140008554882816] - job_utils.py[line:311]: start process command: /data/projects/python/venv/bin/python /data/projects/fate/python/fate_flow/operation/task_executor.py -j 202011200707112994524 -n evaluation_0 -t 202011200707112994524_evaluation_0 -v 0 -r host -p 9999 -c /data/projects/fate/jobs/202011200707112994524/host/9999/evaluation_0/202011200707112994524_evaluation_0/0/task_parameters.json --run_ip 10.244.2.5 --job_server 10.244.2.5:9380 successfully, pid is 22527
[INFO] [2020-11-20 07:17:28,236] [1:140008554882816] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[INFO] [2020-11-20 07:17:28,294] [1:140008554882816] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update successfully
[INFO] [2020-11-20 07:17:28,735] [1:140008554882816] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:28,863] [1:140008554882816] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'party_status': 'running', 'start_time': 1605856648236}
[INFO] [2020-11-20 07:17:29,351] [1:140008554882816] - task_controller.py[line:158]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 on host 9999 executor subprocess start success
[INFO] [2020-11-20 07:17:30,010] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:17:30,068] [1:140008554882816] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:30,499] [22498:139745459410752] - session_utils.py[line:38]: start stop session 202011200707112994524_hetero_secure_boost_0_0_host_9999
[INFO] [2020-11-20 07:17:33,407] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:33,460] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:34,140] [22498:139745459410752] - session_utils.py[line:47]: stop session 202011200707112994524_hetero_secure_boost_0_0_host_9999 success
[INFO] [2020-11-20 07:17:35,877] [22527:140352875779904] - task_executor.py[line:56]: enter task process
[INFO] [2020-11-20 07:17:35,879] [22527:140352875779904] - task_executor.py[line:57]: Namespace(component_name='evaluation_0', config='/data/projects/fate/jobs/202011200707112994524/host/9999/evaluation_0/202011200707112994524_evaluation_0/0/task_parameters.json', job_id='202011200707112994524', job_server='10.244.2.5:9380', party_id=9999, role='host', run_ip='10.244.2.5', task_id='202011200707112994524_evaluation_0', task_version=0)
[INFO] [2020-11-20 07:17:36,419] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[INFO] [2020-11-20 07:17:36,445] [1:140008429041408] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:36,530] [1:140008429041408] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:36,624] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update successfully
[INFO] [2020-11-20 07:17:37,106] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:37,135] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'evaluation_0', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 22527, 'party_status': 'running'}
[INFO] [2020-11-20 07:17:39,881] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:39,965] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:43,191] [1:140008429041408] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:43,256] [1:140008429041408] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:46,439] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:46,507] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:49,535] [1:140008512935680] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:49,618] [1:140008512935680] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:52,015] [22527:140352875779904] - job_tracker.py[line:159]: task id 202011200707112994524_evaluation_0 output data table is none
[INFO] [2020-11-20 07:17:52,041] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[INFO] [2020-11-20 07:17:52,174] [1:140008512935680] - job_saver.py[line:82]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update successfully
[INFO] [2020-11-20 07:17:52,579] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:52,692] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status successfully: {'job_id': '202011200707112994524', 'component_name': 'evaluation_0', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 22527, 'party_status': 'success', 'end_time': 1605856672016, 'elapsed': 16136}
[INFO] [2020-11-20 07:17:52,723] [1:140008554882816] - job_saver.py[line:59]: try to update job 202011200707112994524
[WARNING] [2020-11-20 07:17:52,921] [1:140008554882816] - job_saver.py[line:64]: job 202011200707112994524 update does not take effect: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 87.5, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:17:53,395] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[WARNING] [2020-11-20 07:17:53,450] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update does not take effect
[INFO] [2020-11-20 07:17:53,923] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:53,947] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'component_name': 'evaluation_0', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'run_ip': '10.244.2.5', 'run_pid': 22527, 'party_status': 'success', 'end_time': 1605856672016, 'elapsed': 16136}
[INFO] [2020-11-20 07:17:55,958] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:17:56,019] [1:140008512935680] - job_saver.py[line:72]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status successfully: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_evaluation_0', 'task_version': '0', 'role': 'host', 'party_id': '9999', 'status': 'success'}
[INFO] [2020-11-20 07:17:56,059] [1:140008512935680] - resource_manager.py[line:271]: task 202011200707112994524_evaluation_0 0 return resource successfully
[INFO] [2020-11-20 07:17:56,088] [1:140008512935680] - job_tracker.py[line:424]: clean task 202011200707112994524_evaluation_0 0 on host 9999
[INFO] [2020-11-20 07:18:02,495] [1:140008512935680] - job_tracker.py[line:440]: clean table by namespace 202011200707112994524_evaluation_0_0_host_9999 on host 9999 done
[INFO] [2020-11-20 07:18:02,595] [1:140008512935680] - job_tracker.py[line:446]: clean table by namespace 202011200707112994524_evaluation_0_0 on host 9999 done
[INFO] [2020-11-20 07:18:09,831] [1:140008680724224] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_evaluation_0 host 9999 with success party status process pid:22527
[INFO] [2020-11-20 07:18:09,833] [1:140008680724224] - job_utils.py[line:349]: []
[WARNING] [2020-11-20 07:18:09,834] [1:140008680724224] - job_utils.py[line:382]: this pid 22527 is not job 202011200707112994524 task 202011200707112994524_evaluation_0 host 9999 executor
[INFO] [2020-11-20 07:18:09,847] [1:140008680724224] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_evaluation_0_0_host_9999
[INFO] [2020-11-20 07:18:09,848] [1:140008680724224] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_evaluation_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:09,883] [1:140008680724224] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_evaluation_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22695
[INFO] [2020-11-20 07:18:09,886] [1:140008680724224] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 on host 9999 process 22527 kill success
[INFO] [2020-11-20 07:18:09,886] [1:140008680724224] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:18:09,919] [1:140008680724224] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_evaluation_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:10,301] [1:140008680724224] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[WARNING] [2020-11-20 07:18:10,327] [1:140008680724224] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update does not take effect
[INFO] [2020-11-20 07:18:11,512] [1:140008680724224] - job_saver.py[line:59]: try to update job 202011200707112994524
[INFO] [2020-11-20 07:18:11,579] [1:140008680724224] - job_saver.py[line:62]: job 202011200707112994524 update successfully: {'job_id': '202011200707112994524', 'role': 'host', 'party_id': '9999', 'progress': 100.0, 'src_role': 'guest', 'src_party_id': '10000'}
[INFO] [2020-11-20 07:18:12,159] [1:140008512935680] - job_controller.py[line:283]: job 202011200707112994524 on host 9999 start to save pipeline
[INFO] [2020-11-20 07:18:12,426] [1:140008512935680] - job_tracker.py[line:493]: save 202011200707112994524 model info done. model id: guest-10000#host-9999#model, model version: 202011200707112994524.
[INFO] [2020-11-20 07:18:12,428] [1:140008512935680] - job_controller.py[line:307]: job 202011200707112994524 on host 9999 save pipeline successfully
[INFO] [2020-11-20 07:18:13,066] [1:140008512935680] - job_saver.py[line:43]: try to update job 202011200707112994524 status to success
[INFO] [2020-11-20 07:18:13,123] [1:140008512935680] - job_saver.py[line:46]: update job 202011200707112994524 status successfully
[INFO] [2020-11-20 07:18:13,246] [1:140008512935680] - resource_manager.py[line:172]: return job 202011200707112994524 resource(cores 4 memory 0) on host 9999 successfully, remaining cores: 20 remaining memory: 0
[INFO] [2020-11-20 07:18:13,996] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_evaluation_0 host 9999 with success party status process pid:22527
[INFO] [2020-11-20 07:18:13,998] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_evaluation_0 host 9999 with success party status process pid:22527
[INFO] [2020-11-20 07:18:14,013] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_evaluation_0_0_host_9999
[INFO] [2020-11-20 07:18:14,014] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_evaluation_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:14,055] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_evaluation_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22728
[INFO] [2020-11-20 07:18:14,059] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 on host 9999 process 22527 kill success
[INFO] [2020-11-20 07:18:14,060] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status
[INFO] [2020-11-20 07:18:14,089] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_evaluation_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_evaluation_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:14,472] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_evaluation_0 0
[WARNING] [2020-11-20 07:18:14,495] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_evaluation_0 0 update does not take effect
[INFO] [2020-11-20 07:18:14,911] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 host 9999 with success party status process pid:20527
[INFO] [2020-11-20 07:18:14,912] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 host 9999 with success party status process pid:20527
[INFO] [2020-11-20 07:18:14,942] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_hetero_secure_boost_0_0_host_9999
[INFO] [2020-11-20 07:18:14,943] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_hetero_secure_boost_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:15,001] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_hetero_secure_boost_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22730
[INFO] [2020-11-20 07:18:15,004] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 on host 9999 process 20527 kill success
[INFO] [2020-11-20 07:18:15,005] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status
[INFO] [2020-11-20 07:18:15,041] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_hetero_secure_boost_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:15,414] [22695:140621523359552] - session_utils.py[line:38]: start stop session 202011200707112994524_evaluation_0_0_host_9999
[INFO] [2020-11-20 07:18:15,465] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0
[WARNING] [2020-11-20 07:18:15,494] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_hetero_secure_boost_0 0 update does not take effect
[INFO] [2020-11-20 07:18:15,883] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_intersection_0 host 9999 with success party status process pid:20244
[INFO] [2020-11-20 07:18:15,883] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_intersection_0 host 9999 with success party status process pid:20244
[INFO] [2020-11-20 07:18:15,899] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_intersection_0_0_host_9999
[INFO] [2020-11-20 07:18:15,900] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:16,272] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22737
[INFO] [2020-11-20 07:18:16,274] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 on host 9999 process 20244 kill success
[INFO] [2020-11-20 07:18:16,275] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status
[INFO] [2020-11-20 07:18:16,313] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:16,938] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_0 0
[WARNING] [2020-11-20 07:18:16,966] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_0 0 update does not take effect
[INFO] [2020-11-20 07:18:17,397] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_intersection_1 host 9999 with success party status process pid:19973
[INFO] [2020-11-20 07:18:17,399] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_intersection_1 host 9999 with success party status process pid:19973
[INFO] [2020-11-20 07:18:17,518] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_intersection_1_0_host_9999
[INFO] [2020-11-20 07:18:17,520] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:17,559] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_intersection_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22773
[INFO] [2020-11-20 07:18:17,563] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 on host 9999 process 19973 kill success
[INFO] [2020-11-20 07:18:17,564] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status
[INFO] [2020-11-20 07:18:17,594] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_intersection_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_intersection_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:17,977] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_intersection_1 0
[WARNING] [2020-11-20 07:18:18,006] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_intersection_1 0 update does not take effect
[INFO] [2020-11-20 07:18:18,361] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_dataio_1 host 9999 with success party status process pid:19734
[INFO] [2020-11-20 07:18:18,362] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_dataio_1 host 9999 with success party status process pid:19734
[INFO] [2020-11-20 07:18:18,377] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_dataio_1_0_host_9999
[INFO] [2020-11-20 07:18:18,379] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:18,409] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22776
[INFO] [2020-11-20 07:18:18,412] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 on host 9999 process 19734 kill success
[INFO] [2020-11-20 07:18:18,413] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status
[INFO] [2020-11-20 07:18:18,444] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:18,851] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_1 0
[WARNING] [2020-11-20 07:18:18,888] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_1 0 update does not take effect
[INFO] [2020-11-20 07:18:19,231] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_dataio_0 host 9999 with success party status process pid:19507
[INFO] [2020-11-20 07:18:19,232] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_dataio_0 host 9999 with success party status process pid:19507
[INFO] [2020-11-20 07:18:19,245] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_dataio_0_0_host_9999
[INFO] [2020-11-20 07:18:19,247] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:19,281] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_dataio_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22794
[INFO] [2020-11-20 07:18:19,285] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 on host 9999 process 19507 kill success
[INFO] [2020-11-20 07:18:19,286] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status
[INFO] [2020-11-20 07:18:19,322] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_dataio_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_dataio_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:19,436] [22728:140632582420288] - session_utils.py[line:38]: start stop session 202011200707112994524_evaluation_0_0_host_9999
[INFO] [2020-11-20 07:18:19,614] [22695:140621523359552] - session_utils.py[line:47]: stop session 202011200707112994524_evaluation_0_0_host_9999 success
[INFO] [2020-11-20 07:18:19,699] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_dataio_0 0
[WARNING] [2020-11-20 07:18:19,726] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_dataio_0 0 update does not take effect
[INFO] [2020-11-20 07:18:20,054] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_reader_0 host 9999 with success party status process pid:19259
[INFO] [2020-11-20 07:18:20,055] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_reader_0 host 9999 with success party status process pid:19259
[INFO] [2020-11-20 07:18:20,067] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_reader_0_0_host_9999
[INFO] [2020-11-20 07:18:20,067] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:20,109] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_0_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22819
[INFO] [2020-11-20 07:18:20,112] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_reader_0 0 on host 9999 process 19259 kill success
[INFO] [2020-11-20 07:18:20,113] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0 status
[INFO] [2020-11-20 07:18:20,150] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_0 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_0', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:20,495] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_0 0
[WARNING] [2020-11-20 07:18:20,526] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_0 0 update does not take effect
[INFO] [2020-11-20 07:18:20,902] [1:140008512935680] - job_utils.py[line:374]: try to stop job 202011200707112994524 task 202011200707112994524_reader_1 host 9999 with success party status process pid:18802
[INFO] [2020-11-20 07:18:20,903] [1:140008512935680] - job_utils.py[line:377]: can not found job 202011200707112994524 task 202011200707112994524_reader_1 host 9999 with success party status process pid:18802
[INFO] [2020-11-20 07:18:20,923] [1:140008512935680] - job_utils.py[line:401]: start run subprocess to stop task session 202011200707112994524_reader_1_0_host_9999
[INFO] [2020-11-20 07:18:20,924] [1:140008512935680] - job_utils.py[line:288]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop
[INFO] [2020-11-20 07:18:20,954] [1:140008512935680] - job_utils.py[line:311]: start process command: python3 /data/projects/fate/python/fate_flow/utils/session_utils.py -j 202011200707112994524_reader_1_0_host_9999 --computing EGGROLL --federation EGGROLL --storage EGGROLL -c stop successfully, pid is 22840
[INFO] [2020-11-20 07:18:20,956] [1:140008512935680] - task_controller.py[line:249]: job 202011200707112994524 task 202011200707112994524_reader_1 0 on host 9999 process 18802 kill success
[INFO] [2020-11-20 07:18:20,958] [1:140008512935680] - job_saver.py[line:69]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0 status
[INFO] [2020-11-20 07:18:20,982] [1:140008512935680] - job_saver.py[line:74]: update job 202011200707112994524 task 202011200707112994524_reader_1 0 status update does not take effect: {'job_id': '202011200707112994524', 'task_id': '202011200707112994524_reader_1', 'task_version': 0, 'role': 'host', 'party_id': '9999', 'party_status': 'success'}
[INFO] [2020-11-20 07:18:21,309] [1:140008512935680] - job_saver.py[line:79]: try to update job 202011200707112994524 task 202011200707112994524_reader_1 0
[WARNING] [2020-11-20 07:18:21,336] [1:140008512935680] - job_saver.py[line:84]: job 202011200707112994524 task 202011200707112994524_reader_1 0 update does not take effect
[INFO] [2020-11-20 07:18:22,070] [22728:140632582420288] - session_utils.py[line:47]: stop session 202011200707112994524_evaluation_0_0_host_9999 success
[INFO] [2020-11-20 07:18:23,637] [1:140008512935680] - job_controller.py[line:311]: Job 202011200707112994524 on host 9999 start to clean
[INFO] [2020-11-20 07:18:23,639] [1:140008512935680] - job_controller.py[line:313]: job 202011200707112994524 on host 9999 clean done
